modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -267,110 +268,233 @@
 		<annotation name="@export">
 			<return type="void" />
 			<description>
+				Mark the following property as exported (editable in the Inspector dock and saved to disk). To control the type of the exported property use the type hint notation.
+				[codeblock]
+				@export var int_number = 5
+				@export var float_number : float = 5
```

Comment:
```suggestion
				@export var float_number: float = 5
```

It is [recommended](https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_styleguide.html#declared-types) not to use a space here.

Reply:
Fair, I copied examples from existing docs and they were inconsistent in that regard, so I opted to add a space for clarity.